storage/lab2/pvc-file-silver.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+ name: guestbook-filesilver-pvc
+ labels:
+   billingType: hourly
+   region: us-south
+   zone: dal10
```

Comment:
Should leave out region and zone so that the PVC is created in the same region and zone as the cluster.